packages/flutter/test/foundation/service_extensions_test.dart

```diff
@@ -228,6 +240,12 @@ void main() {
   });
 
   test('Service extensions - debugPaint', () async {
+    final Iterable<Map<String, dynamic>> extensionToggledEvents =
```

Comment:
add a helper method to get the change events for an extension.

Reply:
done.